instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -53,4 +54,12 @@ class CommentSection extends React.Component {
     }
 }
 
+CommentSection.propTypes = {
+    likes: PropTypes.number,
+    comments: PropTypes.array,
+    username: PropTypes.string,
+    text: PropTypes.string,
+    timestamp: PropTypes.string
+}
```

Comment:
Username and text aren't props being sent from the parent container (PostContainer), so you don't need propTypes on those two.